### ISSUE TITLE
discovery-react-components includes all of the @carbon/icons-react package rather than tree-shaking

### ISSUE THREAD

**Issue by psulkava** [watson-developer-cloud/discovery-components#202]
When importing the discovery-react-components into my react app, the package results in the entirety of the @carbon/icons-react package being included in my webpack bundle (5.76 MB) rather than allowing it to be tree shaken out.
I believe its because the discovery-react-components import the icons from the lib path rather than the suggested way like so on line 14 in the SearchInput component.
```
import Search16 from '@carbon/icons-react/lib/search/16';
```
rather than
```
import { Search16 } from '@carbon/icons-react';
```

See this issue for more details on importing carbon icons from the lib path. https://github.com/carbon-design-system/carbon/issues/4847

**Comment by psulkava** [comment]
I was able to fix it in a local fork by changing all of the `@carbon/icons-react` imports and the two ListBox imports from `carbon-components-react` from using the /lib/ path to using the /es/ path.

**Comment by jhpedemonte** [comment]
@psulkava That's great! Thanks for finding this. Would you mind opening up a Pull Request with your changes?

**Comment by psulkava** [comment]
> @psulkava That's great! Thanks for finding this. Would you mind opening up a Pull Request with your changes?

Pull Request has been opened!

**Comment by jhpedemonte** [comment]
Fixed by #209. Let me know if you still see any issues.
